app/models/scoreboard.rb

```diff
@@ -73,6 +77,10 @@ def scored_schools(recent_boundary: 1.month.ago, academic_year: this_academic_ye
 
   private
 
+  def update_name
+    self[:name] = self.name_en
```

Comment:
Just wondering about this and why it's needed. I may have missed the point of something..

Reply:
I think the db name field can probably be removed at some point but currently has a non null constraint so needs to be set on save.  (I figure safer to leave in for now so easier to reverse the changes in this pr if needed)

Reply:
Ahh I see :-)